rules/windows/file_event/file_event_win_susp_powershell_profile.yml

```diff
@@ -0,0 +1,32 @@
+title: PowerShell Profile Modification
+id: b5b78988-486d-4a80-b991-930eff3ff8bf
+status: test
+description: Detects the creation or modification of a powershell profile which could indicate suspicious activity as the profile can be used as a mean of persistence
+author: HieuTT35, Nasreddine Bencherchali
+references:
+    - https://www.welivesecurity.com/2019/05/29/turla-powershell-usage/
+    - https://persistence-info.github.io/Data/powershellprofile.html
+date: 2019/10/24
+modified: 2022/08/24
+logsource:
+    product: windows
+    category: file_event
+detection:
+    selection:
+        TargetFilename|endswith:
+            - '\WindowsPowerShell\Microsoft.PowerShell_profile.ps1'
+            - '\WindowsPowerShell\profile.ps1'
+            - '\PowerShell\Microsoft.PowerShell_profile.ps1'
+            - '\PowerShell\profile.ps1'
+            - '\Windows\System32\WindowsPowerShell\v1.0\Microsoft.PowerShell_profile.ps1'
+            - '\Windows\System32\WindowsPowerShell\v1.0\profile.ps1'
+            - '\Program Files\PowerShell\7\Microsoft.PowerShell_profile.ps1'
+            - '\Program Files\PowerShell\7\profile.ps1'
+    condition: selection
```

Comment:
We need the path?
```yaml
- \profile.ps1
- \Microsoft.PowerShell_profile.ps1
```
many FT ?

Reply:
I will update the rule to reduce the paths where I can